### ISSUE TITLE
Feature: NextCloud compatibility 

### ISSUE THREAD

**Issue by JamborJan** [mnutt/davros#66]
As many of you know there has been a split in the ownCloud team and there is a fork named [Nextcloud](https://nextcloud.com) ([Source Code](https://github.com/nextcloud/)).

The fork is performing quite good and many features have been included recently which have been on the community wish list at ownCloud for years and never have been included.

Davros is compatible with the ownCloud cross platform sync client but unfortunately not with the nextcloud sync client which also has improved since it has been forked.

Right now I'm only wanting to ask if this is something you want to consider or not.

I have tested OS X client which is working well and iOS client which is not working. Error screen below.

![img_1937](https://cloud.githubusercontent.com/assets/7508960/18381507/d94c8dd0-767c-11e6-8302-79c490471a84.PNG)

**Comment by zeigerpuppy** [comment]
I would agree that this is a very desirable feature.  Our users are moving over to NextCloud as we've recommended it as the better supported and more open-featured fork.  It looks like it has more legs than the remaining owncloud client.  They did say, however, that the nextcloud client should retain backwards compatibility, I wonder what is broken.

**Comment by mnutt** [comment]
My understanding is that while the Nextcloud desktop client is a fork of Owncloud, the iOS client is a completely different proprietary client, rebranded from another company. I plan to try to get Nextcloud iOS working, but given that it's no longer free, if you're willing to pay for a client then [Transmit](https://panic.com/transmit-ios/) is a really great option, and one of my favorite iOS apps.

**Comment by JamborJan** [comment]
Thanks for the reply and making the point clear for the iOS nextcloud client, I didn't expect that.

By the way: the nextcloud iOS client as well as the owncloud client are not free on the App Store, they cost 1$.

**Comment by larjona** [comment]
I've tried the NextCloud client for Android (in fact, the stable one in F-Droid and the Beta one also in F-Droid). None of those don't work with Davros, but I think it's due to the same reason the Android OwnCloud app, in its recent versions, does not work either (in recent versions), see #65  
I've tried the oldest version of NextCloud client present in Android (1.0) and it works well.

**Comment by ocdtrekkie** [comment]
I am not sure what you'd need from me to help, but I'd be amazed if it was possible to support Nextcloud's new Windows Mobile client: https://github.com/nextcloud/windows-universal
